README.md

```diff
@@ -5,7 +5,8 @@
 ### Building:
 
 ```bash
-git clone https://github.com/opencontainers/runc
+git clone https://github.com/opencontainers/runc $GOPATH/src/github.com/opencontainers/runc
```

Comment:
This doesn't work for all `GOPATH` setups. I'd recommend `go get -d` to do a download only fetch into the `GOPATH`.